src/extractor/scripting_environment_lua.cpp

```diff
@@ -168,7 +168,10 @@ void LuaScriptingEnvironment::InitContext(LuaScriptingContext &context)
                        &guidance::RoadClassification::SetLowPriorityFlag)
              .property("road_priority_class",
                        &guidance::RoadClassification::GetClass,
-                       &guidance::RoadClassification::SetClass),
+                       &guidance::RoadClassification::SetClass)
+             .property("num_lanes",
+                       &guidance::RoadClassification::GetNumberOfLanes,
+                       &guidance::RoadClassification::SetNumberOfLanes),
```

Comment:
There are road width tags (e.g. http://taginfo.openstreetmap.org/keys/width#combinations) - do you think it could make sense to use those for shortcutting the width calculation via lanes?